custom_components/tahoma/__init__.py

```diff
@@ -3,18 +3,29 @@
 from collections import defaultdict
 from datetime import timedelta
 import logging
+from typing import Any
 
 from aiohttp import CookieJar
 from pyhoma.client import TahomaClient
 from pyhoma.exceptions import BadCredentialsException, TooManyRequestsException
+from pyhoma.models import Command
 import voluptuous as vol
 
 from homeassistant import config_entries
 from homeassistant.components.scene import DOMAIN as SCENE
 from homeassistant.config_entries import ConfigEntry
-from homeassistant.const import CONF_EXCLUDE, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.const import (
+    CONF_EXCLUDE,
+    CONF_PASSWORD,
+    CONF_USERNAME,
+    EVENT_HOMEASSISTANT_STOP,
```

Comment:
Is `EVENT_HOMEASSISTANT_STOP` still used somewhere?

Reply:
nope...